dev-libs/half/half-2.1.0.ebuild

```diff
@@ -0,0 +1,20 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Half-precision floating-point library"
+HOMEPAGE="http://half.sourceforge.net/"
+SRC_URI="https://downloads.sourceforge.net/project/half/half/${PV}/${P}.zip"
```

Comment:
sourceforge `SRC_URI` should use `mirror://sourceforge/` instead.  cf. https://wiki.gentoo.org/wiki//etc/portage/mirrors and portage(5) manpage.